### PR TITLE
CSS Custom Variable

### LINKED ISSUE #991: Frontend : CSS custom properties
Description 📜
I want to work on making documentation on CSS custom properties.
Please /assign this to me.

Domain of Contribution 📊

- [ ] Frontend Dev (HTML, CSS, JS)

### PR DESCRIPTION
**Pull request by smriti2411**
<hr>

## Description 📜

Created a document on CSS Custom Variable along with examples.
Fixes #991
<hr>
- [ ] Documentation (Content Creation in the form of codes or tutorials)
<hr>

## Domain of Contribution 📊
- [x] Frontend Dev HTML/CSS/JS

<hr>

## Checklist ✅
- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

### CODE REVIEW

**Comment by afridihannan** [comment]
Is someone already working on this issue

**Comment by 2024-SANDHYA** [comment]
@code-reaper08 @smriti2411  The whole content is plagiarized, taken from [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties), even the codes are the same.

**Comment by code-reaper08** [comment]
> @code-reaper08 @smriti2411 The whole content is plagiarized, taken from [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties), even the codes are the same.

Thanks for pointing out @2024-SANDHYA !

**Review by praveenscience — CHANGES_REQUESTED** [review]
Can you rephrase or add your own? Seems a lot of content are plagiarised.
